accounts-db/src/transaction_results.rs

```diff
@@ -87,18 +87,16 @@ pub struct TransactionExecutionDetails {
     pub accounts_data_len_delta: i64,
 }
 
+// TODO - remove `DurableNonceFee` as it'd always be Invalid.
```

Comment:
Should have a prequel PR to remove `DurableNonceFee` first? Will that break something else?